docs-ref-conceptual/param-persist-howto.md

```diff
@@ -1,160 +0,0 @@
-| resource_group_name | Execute any command | All Azure CLI references
-| vnet_name | Execute a create command | Azure Web Apps only
-| storage_account_name | Execute a create command |  Azure Web Apps only
-| webapp_name | Execute a create command | Azure Web Apps only
-| function_app_name | Execute a create command | Azure Functions only
-
-## Sample script using persisted parameters
-
-Without persisted parameters, sequential CLI commands must repeat the same parameter values.  With persisted parameters enabled, your stored parameter values can be omitted from sequential commands.  In this example, the `location`, `resource group name` or `storage account name` are repeated in subsequent commands.
-
-```azurecli
-# Reminder: function app and storage account names must be unique.
-
-# turn persisted parameters on
-az config param-persist on
-
-# Create a resource group which will store "resource group" and "location" in persisted parameter.
-az group create --name RGlocalContext --location westeurope
-
-# Create an Azure storage account omitting location and resource group.
-az storage account create \
-  --name sa1localcontext \
-  --sku Standard_LRS
-
-# Create a serverless function app in the resource group omitting storage account and resource group.
-az functionapp create \
-  --name FAlocalContext \
-  --consumption-plan-location westeurope \
-  --functions-version 2
-
-# See the stored parameter values
-az config param-persist show
-```
-
-## Persisted parameter and global variable comparison
-
-There are two Azure CLI commands that can be used to default parameter values: `az config set defaults` and `az config param-persist`.  Use the `az config set defaults.<option>=<value>` command to specify _global variables_ such as group, location, or web. Use `az param-persist` to specify _local default values_ unique to your workload.  Stored values are used by the CLI in place of required arguments.
-
-> [!Important]
-> Persisted parameters override global context values.
->
-
-| Reference | Scope | Set | Use
-|-|-|-|-|
-[`az config set defaults.<option>=<value>`](../latest/docs-ref-autogen/config.yml) | Scoped globally across the CLI | Set explicitly using `az config set defaults.<option>=<value>` | Use for settings such as logging, data collection, and default argument values
-[`az config param-persist`](../latest/docs-ref-autogen/config/param-persist.yml) | Scoped locally to a specific working directory | Set automatically once persisted parameters are turned on | Use for individual workload sequential commands.
-
-### Command examples
-
-Use `az config param-persist` to set a global variable used in the creation of an Azure storage account.
-
-```azurecli
-# set the global variable for resource group
-az config set defaults.group=myGlobalVariableRG
-
-# Create an Azure storage account omitting the resource group relying on the global variable value
-# Substitute the storage account name parameter with a unique value
```

Comment:
The use of `substitute` is not grammatically correct. According to https://www.collinsdictionary.com/dictionary/english/substitute

> If you substitute one thing for another, or if one thing substitutes for another, it takes the place or performs the function of the other thing.

the noun following `substitute` should be the object used for substitution, so the correct expression would be

> Substitute a unique value for the storage account name parameter